play-json/shared/src/test/scala/WritesSharedSpec.scala

```diff
@@ -23,6 +23,16 @@ class WritesSharedSpec extends WordSpec with MustMatchers {
     }
   }
 
+  "Traversable Writes" should {
+    "write Seqs" in {
+      Json.toJson(Seq(5, 4, 3, 2, 1)) mustEqual Json.arr(5, 4, 3, 2, 1)
+    }
+    "write SortedSets" in {
+      import scala.collection.immutable.SortedSet
+      Json.toJson(SortedSet(1, 2, 3, 4, 5)) mustEqual Json.arr(1, 2, 3, 4, 5)
```

Comment:
`immutable.SortedSet` is already ok with master for me, would rather test with a mutable one

Reply:
This test fails on master. I just added another one with the mutable version for completeness though.